lime-cli/src/main/scala/org/bdgenomics/lime/cli/Closest.scala

```diff
@@ -36,7 +36,7 @@ object Closest extends BDGCommandCompanion {
     override var asSingleFile: Boolean = false
     override var deferMerging: Boolean = false
     override var outputPath: String = ""
-    override var disableFastConcat: Boolean = false
+    //override var disableFastConcat: Boolean = false
```

Comment:
oops this was to allow it to build when i was testing sorry abt that will revert